Firmware/ConfigurationStore.cpp

```diff
@@ -251,50 +183,45 @@ static const M500_conf default_conf PROGMEM =
 };
 
 
-static bool is_uninitialized(void* addr, uint8_t len)
+void Config_StoreSettings()
 {
-    while(len--)
-    {
-        if(reinterpret_cast<uint8_t*>(addr)[len] != 0xff)
-            return false;
-    }
-    return true;
+  strcpy_P(cs.version, default_conf.version);
```

Comment:
I feel like this line might be redundant, but I'll leave it here anyway since it doesn't hurt too much